src/features/query-bar/components/filter/FilterButton.js

```diff
@@ -148,18 +164,74 @@ class FilterButton extends React.Component<Props, State> {
         }
     };
 
-    deleteCondition = (index: number) => {
-        const { conditions } = this.state;
+    updateSelectedPrefix = (option: Object) => {
```

Comment:
Can option be something more specific than `Object`?

Reply:
Fixed